### PR TITLE
Web 2.0

### PR DESCRIPTION
**Pull request by ahslr**

### CODE REVIEW

File: web/src/containers/Wallet/MainWallet.js [hunk 1]
```diff
@@ -228,14 +226,6 @@ class Wallet extends Component {
 		this.setState({ activeTab });
 	};
 
-	openContactUs = () => {
```
Comment (abeikverdi) [inline web/src/containers/Wallet/MainWallet.js:231]: @ahslr why did you remove this?

Comment (ahslr) [reply web/src/containers/Wallet/MainWallet.js:231]: The `openContactUs` is passed as a prop to the `assets block` component. Since the logic that triggers a rendering in assets block for that part is removed, I removed the prop as well in the component and stop passing it to the child as well. @abeikverdi

File: web/src/containers/Wallet/AssetsBlock.js [hunk 2]
```diff
@@ -117,18 +114,9 @@ const AssetsBlock = ({
...
 		<div className="wallet-assets_block">
 			<section className="ml-4 pt-4">
-				<EditWrapper stringId="WALLET_DEPOSIT_USD,CONTACT_US_TEXT,WALLET_ESTIMATED_TOTAL_BALANCE">
+				<EditWrapper stringId="WALLET_ESTIMATED_TOTAL_BALANCE">
 					<div className="wallet-search-improvement">
-						{BASE_CURRENCY && IS_XHT ? (
-							<div>
-								{STRINGS.formatString(
-									STRINGS['WALLET_DEPOSIT_USD'],
-									<span className="blue-link pointer" onClick={openContactUs}>
```
Comment (ahslr) [inline web/src/containers/Wallet/AssetsBlock.js:126]: @abeikverdi

File: web/src/containers/Wallet/MainWallet.js (hunk 1 above)
Comment (abeikverdi) [reply web/src/containers/Wallet/MainWallet.js:231]: Ok. looks like a dead code then.

**Review by abeikverdi — APPROVED** [review]
